pkgs/development/libraries/fribidi/default.nix

```diff
@@ -2,15 +2,16 @@
 
 stdenv.mkDerivation rec {
   name = "fribidi-${version}";
-  version = "0.19.7";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "fribidi";
     repo = "fribidi";
-    rev = version;
-    sha256 = "10q5jfch5qzrj2w4fbkr086ank66plx8hp7ra9a01irj80pbk96d";
+    rev = "v${version}";
+    sha256 = "02483nscxc695j9b92clcdf0xb7xkfjry09kqdkkhkzl3vdcj039";
```

Comment:
Maybe try fetchurl with `"https://github.com/fribidi/fribidi/releases/download/v${version}/${name}.tar.bz2"`, it should have the manpage bundled.

Reply:
I don't know how man pages should look in this packaging.
To me it seems like this tarball is the same deal. Build requires `c2man` OR `  mesonFlags = [ "-Ddocs=false" ];` and I see the in `doc` `c2man` input files.

I would elaborate more about the topic in that upstream bugreport fribidi/fribidi#79.

Reply:
You are right. We can probably do without man pages for now.